packages/monorepo-tools/src/monorepo-tools.ts

```diff
@@ -0,0 +1,321 @@
+/* eslint-disable no-console */
+import { spawn } from 'child_process';
+import split2 from 'split2';
```

Comment:
Up to you whether you want to keep this, but these days `split2` (for splitting on line endings) is something like a 8-line utility:

```js
function split2(map: (l: string) => string) {
  const input = new PassThrough();
  return Duplex.from({
    writable: input,
    readable: Readable.from((async function* () {
      for await (const line of /*readline.*/createInterface({ input }))
        yield map(line);
    })())
  });
}
```